website/content/docs/security/acl/tokens/create/create-a-mesh-gateway-token.mdx

```diff
@@ -0,0 +1,364 @@
+---
+layout: docs
+page_title: Create a token for mesh gateway registration
+description: >-
+  Learn how to create ACL tokens that your mesh gateway can present to Consul servers so that they can register with the Consul catalog.
+---
+
+# Create a mesh gateway token
+
+This topic describes how to create a token to enable a mesh gateway to register.
+
+## Introduction
+
+Gateways must present a token linked to policies that grant the appropriate set of permissions. For a mesh gateway to be discoverable and to route to other services in a mesh, the token must be linked to policies that grant the following permissions:
+
+* `mesh:write` to obtain leaf certificates for terminating TLS connections
+* `peering:read` for Consul cluster peering. If you are not using Consul cluster peering, then you can omit the `peering:read` permission.
```

Comment:
Do we need to mention that this is only needed for the mesh gateways that deal with control plane peering traffic?

(Not necessarily here)

I noticed that the non-default partitions don't need this (because they aren't involved in control plane peering traffic, only data plane), but it took me a bit to realize why. Maybe this could be mentioned briefly in the section for non-default partition, rather than in the summary here where it might be a distracting detail?

Reply:
Do you mean that mesh gateways in a non-default partition don't need `peering:read`?

Reply:
Yes

Reply:
Right, then I should remove the `peering:read` permission from the policy in the non-default partition section, also.